src/lib.rs

```diff
@@ -164,6 +171,42 @@
 //! Note that `MousePos` and `MousePosWorld` will no longer be accessible as global resources
 //! -- you can only access them by `Query`ing camera entities.
 //!
+//! ## Opt-out of tracking for cameras
+//!
+//! If you wish to have a camera be excluded from mouse tracking for whatever reason, you may give it the `ExcludeMouseTracking` component.
+//!
+//! ```rust
+//! commands.spawn_bundle(Camera2dBundle::default())
+//!     .insert(ExcludeMouseTracking);
+//! ```
+//!
+//! This camera will not have a `MousePos` or a `MousePosWorld`, as it is completely excluded from mouse tracking.
+//!
+//! One reason to do this is because this crate does not currently support cameras with projections other than Bevy's `OrthographicProjection`. If you use such a camera, even if you don't use it for tracking mouse position, you will find that it panics:
+//!
+//! ```text
+//! thread 'main' panicked at 'only orthographic cameras are supported -- consider adding an ExcludeTracking component: QueryDoesNotMatch(5v0)', src\mouse_pos.rs:148:14
+//! ```
+//!
+//! To get around this, you may choose to have the camera opt-out.
+//!
+//! ```rust
+//! commands
+//!     .spawn_bundle(Camera2dBundle {
+//!         projection: MyCustomProjection,
+//!         ..default()
+//!     })
+//!     .insert(ExcludeMouseTracking);
+//! ```
+//!
+//! If you add the `ExcludeMouseTracking` component to an entity that also has the `MainCamera` component, you will find that the app panics:
+//!
+//! ```text
+//! thread 'main' panicked at 'excluded main camera -- consider removing the ExcludeTracking component from the main camera', src\mouse_pos.rs:216:9
+//! ```
+//!
+//! You should remove either `ExcludeMouseTracking` or `MainCamera`, as they should not be used together.
+//!
```

Comment:
Good note, but this should be in the docs for `ExcludeMouseTracking`, not the README.